sass/nav/_homepage.scss

```diff
@@ -0,0 +1,53 @@
+.nav-homepage {
+  $navImgSrcPrefix: 'sidemenu_';
+  $navHomepageLinksNames: (
+    'home',
+    'barber',
+    'garage',
+    'contact'
+  );
+
+  height: 100%;
+  position: fixed;
+  right: 20px;
+  top: 0;
+  transform-style: preserve-3d;
+
+  .nav-list {
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+
+    li {
+      margin-top: 20px;
+
+      &:first-of-type {
+        margin-top: 0px;
+      }
+
+      a {
+        background-repeat: no-repeat;
+        background-position: center right;
+        display: block;
+        height: 40px;
+        width: 115px;
+        opacity: 0.5;
+        transition: opacity 0.2s;
+
+        @each $name in $navHomepageLinksNames {
+          $linkBackgroundSrc: '/img/' + $navImgSrcPrefix + $name + '.png';
+          &.link-#{$name} {
+            background-image: url($linkBackgroundSrc);
+          }
+        }
+
+        /* Hide the text. */
+        color: transparent;
```

Comment:
With `color: transparent` text is "visible" when selected:
![image](https://cloud.githubusercontent.com/assets/34003/12156864/cf6cb22a-b4cd-11e5-9732-28cc8bd3ef65.png)
You can also hide it like this:
```
overflow: hidden;
text-indent: 100%;
white-space: nowrap;
```